src/layout/Footer/index.css

```diff
@@ -0,0 +1,36 @@
+#footer {
+  width: 100vw;
+  position: absolute;
```

Comment:
if you are positioning absolutly we'd recommend setting `bottom: 0` so that it is always fixed to the bottom. at the moment it is floating